pkg/core/native/invocation_test.go

```diff
@@ -66,7 +66,9 @@ func TestNativeContract_Invoke(t *testing.T) {
 }
 
 func TestNativeContract_InvokeInternal(t *testing.T) {
-	bc, validator, committee := chain.NewMulti(t)
+	bc, validator, committee := chain.NewMultiWithCustomConfig(t, func(c *config.Blockchain) {
+		c.Hardforks = nil
```

Comment:
Why do you need this change here? `ripemd160` is always available in CryptoLib by default.

Reply:
Because in this test there is a case "fail, bad call flag", where it is expected that there are no hardforks.
`// Can't call with these flags, Aspidochelone is active.` 
In other cases, the custom configuration is provided. 
I can change the checked error to `at instruction 1 (SYSCALL): invalid NEF file: invalid Magic` but then its not "bad call flag".

Reply:
> In other cases, the custom configuration is provided

Then let's provide a custom configuration specifically for `fail, bad call flag` test-case. But in general for basic usage, the default configuration should be used.